Source/BestProjectEver/Public/Weapons/BPE_Weapon.h

```diff
@@ -276,6 +284,20 @@ class BESTPROJECTEVER_API ABPE_Weapon : public AActor, public IBPE_InteractWithC
 	UFUNCTION(NetMulticast, Unreliable, WithValidation)
 	void Multicast_PlayImpactFireEffects(const FVector& ImpactPoint);
 
+	//------------------------------------------------------------------------------------------------------------------
+	//Ammo
+
+	/** [server] update current weapon ammo
+	 * @param Ammo ammunition reloaded or removed to current weapon
+	 */
+	void UpdateAmmo(const int32 Ammo = -1);
```

Comment:
I don't think a default value of -1 for ammo should be used here. Seeing a call to UpdateAmmo() doesn't look like you are actually specifying the amount in the function call. I think it would be clearer if you had to use the -1 with each call when you need to.

Reply:
![image](https://user-images.githubusercontent.com/85327912/176239987-72a335af-f687-4c4b-ae6d-8f7ee7f9f3d1.png)
![image](https://user-images.githubusercontent.com/85327912/176240118-25d6c00b-cb88-41c3-8ab6-5f5fca0675ad.png)